.gitignore

```diff
@@ -5,3 +5,4 @@ docs
 .nyc_output/
 *.lcov
 dist
+*package-lock.json
```

Comment:
I believe we need the `package-lock.json` file to keep track of the versions of the packages we use, and it is also one of the files that is modified when we are preparing for a new release. Is there a reason for adding it to `.gitignore`?